packages/clay-multi-step-nav/src/Item.tsx

```diff
@@ -19,36 +19,37 @@ export interface IProps extends React.HTMLAttributes<HTMLLIElement> {
 
 	/**
 	 * Flag to indicate if `complete` classname should be applied
+	 * @deprecated since v3.91.0 - use `state` instead.
 	 */
 	complete?: boolean;
 
 	/**
-	 * Flag to indicate that the step has an error.
+	 * Flag to indicate if progress line should expand out from step
 	 */
-	error?: boolean;
+	expand?: boolean;
 
 	/**
-	 * Flag to indicate if progress line should expand out from step
+	 * Defines the status of the current step.
```

Comment:
It's probably more clear to just say "Defines the status of the step." removing "current". Since we are inside the step/item component this step may not always be the "current" step in the multistepnav.